### PR TITLE
Adapt to refactored DropMessage payload

### PR DESCRIPTION
**Pull request by L-Henke**
Adapt to https://github.com/Qabel/qabel-core/pull/376

### CODE REVIEW

File: src/main/java/de/qabel/helloworld/QblHelloWorldModule.java [hunk 1]
```diff
@@ -74,17 +75,15 @@ public void moduleMain() {
...
+			DropMessage dm = new DropMessage(contact.getContactOwner(),
+					"Hello this is " + contact.getContactOwner().getAlias(), HELLO_WORLD_TYPE);
 			DropActor.send(EventEmitter.getDefault(), dm, contact);
 		}
 	}
 
 	@Override
 	public void onEvent(String event, MessageInfo info, Object... data) {
-		// Handle Contact changed and DropMessage received events
+		// Handle Ccd .ontact changed and DropMessage received events
```
Comment (jasinai) [inline src/main/java/de/qabel/helloworld/QblHelloWorldModule.java]: I think you don't want to have this extra `cd .` in the comment. :)

Comment (L-Henke) [inline src/main/java/de/qabel/helloworld/QblHelloWorldModule.java]: Definitely!

**Comment by thechauffeur** [comment]
Wait until the discussion in Qabel/qabel-core#376 is resolved.

**Comment by jasinai** [comment]
The discussion is resolved long ago, so I will merge this PR and restart the travis build for Qabel/qabel-core#376.

**Pull request merged**